docs/reference/cat/alias.asciidoc

```diff
@@ -51,6 +51,8 @@ include::{es-ref-dir}/rest-api/common-parms.asciidoc[tag=cat-v]
 
 include::{es-ref-dir}/rest-api/common-parms.asciidoc[tag=expand-wildcards]
 
+include::{es-ref-dir}/rest-api/common-parms.asciidoc[tag=master-timeout]
```

Comment:
Not sure if these docs and rest-api-spec changes are out of scope or not for this PR.

Reply:
Seems ok to me, tho note that we have never before supported `?master_timeout` on these APIs, this isn't reinstating a parameter that previously was removed.

Reply:
Yeah I realized that, but I figured it made sense to add them if they're parameters we're accepting.